bin/oneoff/diff_email_previews

```diff
@@ -16,21 +16,22 @@ To diff emails in 2 folders in the current directory: './diff_email_previews dif
 }
 
 # List of email previews are copied from
-# http://localhost-studio.code.org:3000/rails/mailers/pd/workshop_mailer/
+# http://localhost-studio.code.org:3000/rails/mailers/pd_workshop_mailer/
 PREVIEWS = %w(
   detail_change_notification__admin
   detail_change_notification__csf_deepdive
   detail_change_notification__csf_intro
-  exit_survey__csd_1
```

Comment:
I'm wondering if those emails had to do with TeacherCon since they are next to it in the list. Be interested to know what the email associated with it said

Reply:
I'm wondering if this oneoff script should even be updated ... I can restore this file if you think it's best to leave it untouched (no mentions of it in Slack)

Reply:
Seems like if it has not been on staging for a while it's probably fine to go ahead without it.